FileProviderExtensionUI/ReauthenticationViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ReauthenticationViewController.swift
+//  Cryptomator
+//
+//  Created by Majid Achhoud on 08.10.24.
+//  Copyright © 2024 Skymatic GmbH. All rights reserved.
+//
+
+import CryptomatorCommonCore
+import UIKit
+
+class ReauthenticationViewController: UITableViewController {
```

Comment:
I'd suggest renaming this class to `UnauthorizedErrorViewController`.

Reply:
Renamed it in commit 99a4534.